doc/tutorials/README.md

```diff
@@ -1,6 +1,16 @@
 # Tutorials
-Guides through different use scenarios.    
-     
-[Run application] - More detailed version of Getting Started        
-[Run application with sample data](./run-application-with-sample-data.md) - How to run application with sample data, what kind of sample data do we provide and which functionality is presented with them       
-[Run application with own data](./run-application-with-own-data.md) - How and where to add another data set into the application      
\ No newline at end of file
+Guides through different use scenarios and modifications.    
+
+## Use
+
+How to use application and make small configuration settings.
+
+- [Quick start with Docker](./quick-start.md) 
+
+## Advanced
+
+More advanced changes, requiring development setup.
+
+- [Run application locally for development] - More detailed version of Getting Started
+- [Run application with sample data](./advanced/run-application-with-sample-data.md) - How to run application with sample data, what kind of sample data do we provide and which functionality is presented with them
+- [Run application with own data](./advanced/run-application-with-own-data.md) - How and where to add another data set into the application
```

Comment:
dot

Reply:
not sure they should be at the end of bullet list items

Reply:
or we can use small letter in the beginning. We already have dots in the end in main README.md.